### PR TITLE
docs: add doc to .all-contributorsrc for aharris88

### PR DESCRIPTION
**Pull request by aharris88**

### CODE REVIEW

**Comment by kentcdodds** [comment]
Do you mind regenerating the README? `npm run generate-contributors`

I'd love to get a bot for this kind of thing in the future!

**Comment by aharris88** [comment]
@kentcdodds That script doesn't exist...

**Comment by kentcdodds** [comment]
oh, sorry, I meant `npm start generate-contributors`.

**Comment by aharris88** [comment]
Ok, done.

File: README.md [hunk 1]
```diff
@@ -141,8 +141,6 @@ This will be forwarded onto `prettier-eslint` as (`sillyLogs`) and is useful for
 Thanks goes to these people ([emoji key][emojis]):
 
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
-| [<img src="https://avatars.githubusercontent.com/u/1500684?v=3" width="100px;"/><br /><sub>Kent C. Dodds</sub>](https://kentcdodds.com)<br />[💻](https://github.com/kentcdodds/prettier-eslint-cli/commits?author=kentcdodds) [📖](https://github.com/kentcdodds/prettier-eslint-cli/commits?author=kentcdodds) 🚇 [⚠️](https://github.com/kentcdodds/prettier-eslint-cli/commits?author=kentcdodds) | [<img src="https://avatars3.githubusercontent.com/u/3266363?v=3" width="100px;"/><br /><sub>Adam Harris</sub>](https://github.com/aharris88)<br />👀 | [<img src="https://avatars.githubusercontent.com/u/622118?v=3" width="100px;"/><br /><sub>Eric McCormick</sub>](https://ericmccormick.io)<br />👀 |
```
Comment (kentcdodds) [inline README.md]: huh... This is weird. Shouldn't have done that... Let me give this a look later. Sorry about all this!

Comment (aharris88) [inline README.md]: Oh yeah, that's not good...

Comment (aharris88) [inline README.md]: That's fixed now. I was missing a comma in `.all-contributorsrc`.

**Comment by kentcdodds** [comment]
Awesome! Thank you!

**Pull request merged**